src/components/FeaturePanel/Runways/Runways.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useQuery } from 'react-query';
+import { Typography } from '@mui/material';
+import { useFeatureContext } from '../../utils/FeatureContext';
+import { loadRunways } from './loadRunways';
+import { SuccessRunways } from './SuccessRunways';
+import { DotLoader } from '../../helpers';
+
+const RunwaysInner = () => {
+  const { feature } = useFeatureContext();
+  const { id } = feature.osmMeta;
```

Comment:
Please always use type + id, in osm there could be node/1 relation/1 and way/1 in the same time. I suggest forwarding the whole `feature.osmMeta` as type `ApiId`